documentation/modules/deploying/proc-using-access-operator.adoc

```diff
@@ -129,4 +129,23 @@ data:
 
 . Make the secret available to your application. 
 + 
-Applications can reference the secret directly, or you can inject it into an application's environment using an operator to implement the link:https://servicebinding.io/spec/core/1.0.0/[Service Binding specification].
\ No newline at end of file
+Applications can reference the secret directly, or you can inject it into an application's environment using an operator to implement the link:https://servicebinding.io/spec/core/1.0.0/[Service Binding specification].
+
+A user has the option for specifying the secret name if desired. This is achieved by adding `SecretName` field to the `KafkaAccess` spec.
```

Comment:
```suggestion
A user has the option for specifying the secret name if desired. This is achieved by adding `secretName` field to the `KafkaAccess` spec.
```

Reply:
@scholzj Yes, this is a bit premature. The following needs to be merged first:
https://github.com/strimzi/kafka-access-operator/pull/90
But are you saying we should not add this feature until the next release of Kafka Access Operator with v5 of JOSDK?

Reply:
When this PR is merged, this change would automatically go into the next Strimzi release. But the change will be actually available only after the Access Operator is released and the new release is added to this repository. Otherwise it would mislead users into assuming the Access Operator has features it does not have just yet. So we cannot merge it before that happens and having the PR open makes not much sense until then.

This has nothing to do with https://github.com/strimzi/kafka-access-operator/pull/90. That can be of course merged whenever it is ready. It is just the docs change in this repository that has to be done only after the Access Operator release and not after the https://github.com/strimzi/kafka-access-operator/pull/90 PR is merged.

Reply:
Thanks for Clarification.